api/SourceBuffer.json

```diff
@@ -446,14 +446,41 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/SourceBuffer/audioTracks",
           "support": {
             "chrome": {
-              "version_added": "23"
+              "version_added": "51",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
             },
             "chrome_android": {
-              "version_added": "33"
-            },
-            "edge": {
-              "version_added": "12"
+              "version_added": "51",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "enable-experimental-web-platform-features",
+                  "value_to_set": "enabled"
+                }
+              ]
```

Comment:
This doesn't seem like it should be here.

Reply:
Can you clarify what's wrong? The important bit is that these APIs aren't shipped in any Chromium based browser. Because of the presence of some of the flag data around this, I'm having to represent it has a flagged entry for consistency, rather than just setting `version_added` to false.

Reply:
Sorry. This is pre-Chromium edge, but it uses the name of a Chromium flag. This could be confusing.

Reply:
The line you've commented on is in the chrome_android block, and in the Edge block below the flag is on the Chromium-based Edge, version 79.

If I'm still missing it, can you comment on the exact line you'd like to see changed?